### ISSUE TITLE
SA1125 UseShorthandForNullableTypes incorrectly reported for member in XML comment

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#638]
The following expression incorrectly results in SA1125:

```csharp
/// <seealso cref="Nullable{T}.Value"/>
```
